lib/build-tokens.js

```diff
@@ -56,9 +57,14 @@ async function buildTokensCommand(commandArgs) {
     source: tokensSource
       ? [`${tokensSource}/core/**/*.json`, `${tokensSource}/core/**/*.toml`]
       : [],
+    preprocessors: ['annotate-tokens-with-references', 'tokens-studio'],
```

Comment:
[inform] See docs on [preprocessors](https://styledictionary.com/reference/hooks/preprocessors/). `annotate-tokens-with-references` is custom.

See `sd-transforms` README for more information on the [`tokens-studio` preprocessor](https://github.com/tokens-studio/sd-transforms?tab=readme-ov-file#using-the-preprocessor).